docs/modules/theme/pages/table.adoc

```diff
@@ -14,6 +14,7 @@ The keys in the `table` category control the arrangement and style of tables and
 |background-color
 |xref:color.adoc[Color] +
 (default: `transparent`)
+//TODO Where is the transparent value for the table background set in the base theme?
```

Comment:
Technically, it's not. It's just effectively transparent. Whenever I write `transparent` as a default background color value, this is usually what I mean. That it's not inherited...it's just not set. But we could just write `not set` here.